tests/test_notebook.py

```diff
@@ -43,3 +43,18 @@ def test_from_json(from_json):
     demand_max_flow = get_node_attribute(demand, "max_flow")
 
     assert demand_max_flow["value"] == "demand_max_flow - AggregatedParameter"
+
+
+def test_d3_data():
+
```

Comment:
Just add a quick description for this. Comment that it only checks nodes are the same and number of links are the same.